File: Ingeniero.java
package Trabajo;

public class Ingeniero {
    public String nombre;

    public String genero;

    public String peso;

    public Ingeniero(String nom, String gen, String pes)
    {
        establecerNombre(nom);
        establecerGenero(gen);
        establecerPeso(pes);
    }

    //nombre
    public String obtenerNombre()
    {
        return nombre;
    }

    public void establecerNombre(String nom)
    {
        nombre = nom;
    }

    //genero
    public String obtenerGenero()
    {
        return genero;
    }

    public void establecerGenero(String gen)
    {
        genero=gen;
    }

    //edad
    public String obtenerPeso()
    {
        return peso;
    }
    public void establecerPeso(String pes)
    {
        peso = pes;
    }
}

File: ingeniero.php
<?php

namespace App\Model;

class Ingeniero
{
    public $nombre = "";
    public $edad = "";
    public $telefono = "";

    //nombre
    public function __construct($nom)
    {
       $this->establecerNombre();
    }

    public function obtenerNombre()
    {
        return $this->nombre;
    }

    public function establecerColor($nom)
    {
        $this->nombre = $nom;
    }

    //edad
    public function __construct($ed)
    {
       $this->establecerEdad();
    }

    public function obtenerEdad()
    {
        return $this->edad;
    }

    public function establecerEmail($ed)
    {
        $this->edad = $ed;
    }

    //telefono
    public function __construct($tel)
    {
       $this->establecerTelefono();
    }

    public function obtenerTelefono()
    {
        return $this->telefono;
    }

    public function establecerMarca($ed)
    {
        $this->telefono = $ed;
    }
}